packages/react/src/FrostedGlassVFX/FrostedGlassVFX.module.css

```
@@ -0,0 +1,63 @@
+.FrostedGlassVFX {
+  position: relative;
+  background-color: var(--brand-FrostedGlassVFX-bgColor);
+}
+
+.FrostedGlassVFX--rounded-corners {
+  border-radius: var(--brand-borderRadius-xlarge);
+}
+
+@supports (backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low))) or
+  (-webkit-backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low))) {
+  .FrostedGlassVFX--intensity-low {
+    backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low));
+  }
+
+  .FrostedGlassVFX--intensity-medium {
+    backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-medium));
+  }
+
+  .FrostedGlassVFX--intensity-high {
+    backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-high));
+  }
+}
+
+@supports not (backdrop-filter: blur(var(--brand-FrostedGlassVFX-blurIntensity-low))) {
+  .FrostedGlassVFX--intensity-low::after,
+  .FrostedGlassVFX--intensity-medium::after,
+  .FrostedGlassVFX--intensity-high::after {
+    position: absolute;
+    inset: 0;
+    content: '';
+    background-color: var(--brand-color-canvas-default);
+    z-index: 0;
+    border-radius: var(--brand-borderRadius-xlarge);
+  }
+
+  .FrostedGlassVFX > * {
+    z-index: 1;
+  }
+}
+
+.FrostedGlassVFX::before {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  inset: 0;
+  padding: var(--brand-borderWidth-thin);
+  background: var(--brand-FrostedGlassVFX-borderMask);
+  mask: linear-gradient(var(--base-color-scale-black-0) 0 0) exclude,
+    linear-gradient(var(--base-color-scale-black-0) 0 0) content-box;
+}
+
+.FrostedGlassVFX--rounded-corners::before {
+  border-radius: var(--brand-borderRadius-xlarge);
+}
+
+.FrostedGlassVFX > * {
+  background-color: transparent;
+}
+
+.FrostedGlassVFX > *::before {
+  border-color: transparent;
+}
```

```suggestion
.FrostedGlassVFX > *
.FrostedGlassVFX > *::before {
  border-color: transparent;
}
```

We may also want to override the border if it is set in the element itself rather than in the `::before` pseudo-element.